README.md

```diff
@@ -72,6 +72,8 @@ then you can `prepareForUse` manually:
 Haptic.selection.prepareForUse()
 ```
 
+**Note:** Haptics should only be generated and prepared for reuse on the main thread.
```

Comment:
Can we add `>` to the start of this line to make it stand out a bit more?

Reply:
@KaneCheshire Done 🙂 it's slightly greyed out but does stand out a bit more which is good – what do you think?